code/modules/mapping/space_management/traits.dm

```diff
@@ -51,7 +51,7 @@
 
 /// Attempt to get the turf below the provided one according to Z traits
 /datum/controller/subsystem/mapping/proc/get_turf_below(turf/T)
-	if (!T || !initialized)
+	if (!T || !multiz_levels)
```

Comment:
this wont be true because multiz_levels will be an empty list before its populated which is truthy. do
```suggestion
	if (!T || !length(multiz_levels))
```